cmd/monitor/monitor_enable.go

```diff
@@ -0,0 +1,53 @@
+package monitor
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	sts "gitlab.com/stackvista/stackstate-cli2/generated/stackstate_api"
+	stscobra "gitlab.com/stackvista/stackstate-cli2/internal/cobra"
+	"gitlab.com/stackvista/stackstate-cli2/internal/common"
+	"gitlab.com/stackvista/stackstate-cli2/internal/di"
+)
+
+func MonitorEnableCommand(cli *di.Deps) *cobra.Command {
+	args := &IdArgs{}
+	cmd := &cobra.Command{
+		Use:   "disable",
+		Short: "Disable a monitor",
+		Long:  "Disable a monitor.",
```

Comment:
All these fields have incorrect messages. The function name is `MonitorEnableCommand`.

Reply:
Fixing it